<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>599</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Desktop control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QPushButton" name="connectPushButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>390</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ipTextEdit">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>340</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:17pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="ipLabel">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>290</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите ip партнера</string>
      </property>
     </widget>
     <widget class="QPushButton" name="waitPushButton">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>480</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Разрешить подключение</string>
      </property>
     </widget>
     <widget class="QLabel" name="myIPLabel">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>220</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="controller_page"/>
    <widget class="QWidget" name="controlled_page">
     <widget class="QLabel" name="controlledLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>301</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>Нас контролируют!!!!</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="stopSharingPushButton">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>530</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Отключиться</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
